docs/section6/Rule6-9.md

```diff
@@ -11,22 +11,62 @@
 
   1. Building total area is more than 5,000sq.ft.  
 
-**Manual Check:**  
+**Manual Check:** Yes  
 
-  1. Where building total area is more than 5,000sq.ft., manual review is required to verify automatic shutoff controls are modeled correctly in the Baseline.  
+**Function Call:**  
 
-**Function Call:** None  
+  - compare_schedules()
+  - normalize_space_lighting_schedule()
 
 **Evaluation Context:** Each Data Element  
 **Data Lookup:** None  
 
 ## Rule Logic: 
 
-- For each building_segment in the Baseline model: ```for building_segment_b in B_RMR.building.building_segments:```  
-  - For each thermal_block in building segment: ```for thermal_block_b in building_segment.thermal_blocks:```  
-    - For each zone in thermal block: ```for zone_b in thermal_block_b.zones:```  
-      - For each space in zone: ```for space_b in zone.spaces:```  
-        - Add space floor area to building total floor area: ```building_total_area_b += space_b.floor_area```  
-- **Applicability Check 1:**```if building_total_area_b > 5000:```  
+- For each building_segment in the Baseline model: `for building_segment_b in B_RMR.building.building_segments:`  
+
+  - For each thermal_block in building segment: `for thermal_block_b in building_segment.thermal_blocks:`  
+
+    - For each zone in thermal block: `for zone_b in thermal_block_b.zones:`  
+
+      - For each space in zone: `for space_b in zone.spaces:`  
+
+        - Add space floor area to building total floor area: `building_total_area_b += space_b.floor_area`  
+
+- **Applicability Check 1:**`if building_total_area_b > 5000:`  
+
+- Get building open schedule in the baseline model: `building_open_schedule_b = B_RMR.building.building_open_schedule`  
+
+- For each building segment in building: `building_segment_b in B_RMR.building.building_segments:`  
+
+  - For each thermal block in building segment: `thermal_block_b in building_segment_b.thermal_blocks:`  
+
+    - For each zone in thermal block: `zone_b in thermal_block_b.zones:`  
+
+      - For each space in zone: `space_b in zone_b.spaces:`  
+
+        - Get normalized space lighting schedule: `normalized_schedule_b = normalize_space_lighting_schedule(space_b)`  
+
+        - Get matching space in P_RMR: `space_p = match_data_element(P_RMR, Spaces, space_b.id)`  
+
+          - Get normalized space lighting schedule in P_RMR: `normalized_schedule_p = normalize_space_lighting_schedule(space_p)`
+
+        - Check if automatic shutoff control is modeled in space during unoccupied hours (i.e. if lighting schedule hourly value in B_RMR is equal to P_RMR during unoccupied hours): `schedule_comparison_result = compare_schedules(normalized_schedule_b, normalized_schedule_p, building_open_schedule_b, none)`  
```

Comment:
we should clarify we mean by 'unoccupied hrs'. Hrs where building open schedule = 0 would be 'unoccupied hrs'.
we should also probably rename 'unoccupied' to 'open' or 'closed'. Occupancy schedule is differant from the building open schedule, so this should probably say 'check if automatic shutoff controls modeled during building closed hrs'

Reply:
Sounds like comparison factor should be a number instead of "none" for . E.g., some flag like - 101.

Reply:
Right, in this case we will not be using the comparison factor. The mask schedule needs to be 2 for all building open hours to skip the comparison. I'll put -111 for now and leave it up to the actual code.

Reply:
"Unoccupied hours" is now changed to "building closed hours".